controllers/certManager/cert_issuer.go

```diff
@@ -0,0 +1,71 @@
+//
```

Comment:
maybe change the file name to `cert_resource.go` or `cert_cr.go` since it's not just issuers

Reply:
Nice catch!
Previously I thought `cert_issuer` is `certificate_issuer`. But it will cause misunderstanding.
Updated!